### PR TITLE
chore: create initial draft of documentation in README.md

### PR DESCRIPTION
**Pull request by mi-xu**
This PR adds some basic documentation that explains how to use the functions provided by this library.

### CODE REVIEW

**Comment by mi-xu** [comment]
Here is a link to a rendered version of the markdown: https://hackmd.io/@jmPpL1ukSO6T9klrhuvriA/ryW_ldCqm

File: README.md [hunk 1]
```diff
@@ -2,8 +2,166 @@
...
-`createPresentation()`
\ No newline at end of file
+```
+npm install did-jwt-vc
+```
+
+Using Yarn:
+
+```
+yarn add did-jwt-vc
```
Comment (mirceanis) [inline README.md]: we probably don't need to specify `yarn`

File: README.md [hunk 2]
```diff
@@ -2,8 +2,166 @@
...
+
+#### Creating a Verifiable Credential
+
+Specify a `payload` matching the `VerifiableCredentialPayload` interface. Create a JWT by signing it with the previously configured `issuer` using the `createVerifiableCredential` function:
+
+```typescript
+import { VerifiableCredentialPayload, createVerifiableCredential } from 'did-jwt-vc'
+
+const vcPayload: VerifiableCredentialPayload = {
+  sub: '0x435df3eda57154cf8cf7926079881f2912f54db4',
```
Comment (mirceanis) [inline README.md]: example subject should also be a DID

File: README.md (hunk 1 above)
Comment (mi-xu) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (mi-xu) [reply README.md]: fixed

**Pull request merged**